### ISSUE TITLE
Tabnine running even after closing ide

### ISSUE THREAD

**Issue by mukulnag09** [codota/tabnine-intellij#296]
I have seen this occurs on multiple occasions . 
Even after closing all ides or vs code , i still see tabnine exe running in the task manager , even after waiting for some time the application dont auto close.

**Comment by maya-tabnine** [comment]
Hi I tried to reproduce but couldn't. For me, it stopped immediately. 
Are you sure no idea is still open? Would you restart your machine and check again by open+close an idea? 
It shouldn't keep running.

**Comment by mukulnag09** [comment]
![IMG20220326220257](https://user-images.githubusercontent.com/69966779/160249114-0527f7b4-3970-4835-b52a-9a62f7211566.jpg)
After  multiple hours of not using any ide tabnine was running for no reason at all

**Comment by mukulnag09** [comment]
Yes, I had the plugin installed on both vs cod and  IntelliJ IDE, but this occurred after a few hours of inactivity and not using any of the ide in that time frame.
And it persisted till I manually closed it using task manager and as u can see in the above image no IDE was opened.
I will try and uninstall it from one of the IDE and then try and see if it persists .

**Comment by luannguyenbbv** [comment]
Hi,   
I see it still running on my Laptop when I already close IntelliJ IDEA.
![image](https://github.com/codota/tabnine-intellij/assets/139457427/fb6d8d75-48bf-40f2-b7b8-6798428b31ec)

**Comment by luanldt** [comment]
Hi,   
I see it still running on my Laptop when I already close IntelliJ IDEA.
![image](https://github.com/codota/tabnine-intellij/assets/139457427/fb6d8d75-48bf-40f2-b7b8-6798428b31ec)
